lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -174,11 +173,7 @@ WasmBinaryReader::ReadSectionHeader()
 
 #if ENABLE_DEBUG_CONFIG_OPTIONS
     Assert(idSize < 64);
-    char16 buf[64];
-    size_t convertedChars = 0;
-    mbstowcs_s(&convertedChars, buf, idSize + 1, sectionName, _TRUNCATE);
-    buf[idSize] = 0;
-    TRACE_WASM_SECTION(_u("Section Header: %s, length = %u (0x%x)"), buf, sectionSize, sectionSize);
+    TRACE_WASM_SECTION(_u("Section Header: %s, length = %u (0x%x)"), sectionName, sectionSize, sectionSize);
```

Comment:
I'm not sure if `%S` will work here to print `char` string. If that doesn't, try use utf8::NarrowToWide